### DISCUSSION TITLE
Collection Item Duplication

### DISCUSSION THREAD

**Discussion by ernst86913** [directus/directus#7594] [Archive]
Hi -
can anybody help me with the "Item duplication" feature which can be set on collections?

Whats is it? How does it work?

I can only find the following infos in the docu:
a) https://docs.directus.io/guides/fields/#duplicating-a-field
... seems to be out-dated / wrong
b) https://docs.directus.io/guides/collections/#configuring-a-collection
... but this just link to above without any addtl. infos

Thanks in advance,
Roman

**Comment by rijkvanzanten** [comment]
It controls what fields are copied when using the "Save as Copy" option when editing an item. By checking on/off specific fields, you can control what data is copied over to the new item, and what things are ignored. With the fields being nested, you can control things like "When copying an article, use the existing author" vs "When copying an article, also create a new author using the existing name field on the author" etc

(cc @jacobrienstra for docs later)

**Comment by ernst86913** [comment]
Thanks a lot for clarification. Helps a lot.


Von meinem iPhone gesendet
